### ISSUE TITLE
cls_token during masked reconstruction pretraining

### ISSUE THREAD

**Issue by etaoxing** [Pang-Yatian/Point-MAE#20]
Why is cls_token only used during finetuning in the `PointTransformer` module, but not in the `Point_MAE` module during masked reconstruction pretraining?

https://github.com/Pang-Yatian/Point-MAE/blob/61727f76e9d0c28babf422505073bd43c2f517bc/models/Point_MAE.py#L531-L548

Point-BERT appends the cls_token when training masked reconstruction (see [link](https://github.com/lulutang0608/Point-BERT/blob/980c1e8f2148191b0e9157efdc6ba157f00e893b/models/Point_BERT.py#L380-L415), and also Figure 3 of their paper).

**Comment by Pang-Yatian** [comment]
Hi,

There is no particular reason. Empirically, we find adding a cls_token could make fine-tuning more stable.
